mlflow/langchain/__init__.py

```diff
@@ -542,6 +543,11 @@ def load_retriever(persist_directory):
         streamable: A boolean value indicating if the model supports streaming prediction. If
             True, the model must implement `stream` method. If None, If None, streamable is
             set to True if the model implements `stream` method. Default to `None`.
+        resources: A list of model resources or a resources.yaml file containing a list of
```

Comment:
follow-up: do we have documentation in OSS+Databricks for how to construct a value for this parameter?

Reply:
Yes we have it here: https://docs.databricks.com/en/generative-ai/log-agent.html#pyfunc-resources and here: https://docs.databricks.com/en/generative-ai/deploy-agent.html#deploy-an-agent-using-deploy

Reply:
Great! Can we add a link to either doc in docstring?

Reply:
Yes, these are more for pyfunc doc strings. Will create a ticket to write docs for langchain and update that here :)